modules/data-model/forms/impl/src/main/java/io/uhndata/cards/forms/internal/serialize/labels/FileLabelProcessor.java

```diff
@@ -42,6 +42,12 @@
 @Component(immediate = true)
 public class FileLabelProcessor extends SimpleAnswerLabelProcessor implements ResourceJsonProcessor
 {
+    @Override
+    public String getDescription()
+    {
+        return super.getDescription() + "Get the human readable answer for file questions.";
```

Comment:
Is this what we're doing? If not, please provide the correct details.
```suggestion
        return super.getDescription() + "Get the human readable answer for file questions by outputting the file name instead of the path.";
```